### PR TITLE
Update lan_form_handler.php

### PR DESCRIPTION
**Pull request by yesszus**
Add pharses.

### CODE REVIEW

File: e107_handlers/form_handler.php [hunk 1]
```diff
@@ -1024,11 +1024,11 @@ function avatarpicker($name,$curVal='',$options=array())
 		if($count == 0)
 		{
 			$text .= "<div class='row'>";
-			$text .= "<div class='alert alert-info'>No Avatars Available</div>"; //TODO LAN
+			$text .= "<div class='alert alert-info'>".LAN_EFORM_005."</div>"; 
 
 			if(ADMIN)
 			{
-				$text .= "<div class='alert alert-danger'>Admin-Only Notice: The folder <b>".e_AVATAR_DEFAULT."</b> is empty. Upload some default avatars images to this folder for users to choose avatars from.</div>"; //TODO LAN
+				$text .= "<div class='alert alert-danger'>".LAN_EFORM_006." <b>".e_AVATAR_DEFAULT."</b> ".LAN_EFORM_007."</div>";
```
Comment (CaMer0n) [inline e107_handlers/form_handler.php]: Please use lanVars. (ie. never split a sentence with LANs) 
"Admin-Only Notice: The folder [x] is empty. Upload some default avatars images to this folder for users to choose avatars from."

Comment (yesszus) [inline e107_handlers/form_handler.php]: Edited

**Comment by yesszus** [comment]
Work for me without errors.

**Comment by CaMer0n** [comment]
Thank you!

File: e107_admin/comment.php [hunk 2]
```diff
@@ -73,15 +73,15 @@ class comments_admin_ui extends e_admin_ui
 	
 	   		'comment_type' 			=> array('title'=> LAN_TYPE,			'type' => 'method',			'width' => '10%',  'filter'=>TRUE),	
 			
-			'comment_item_id' 		=> array('title'=> "item id",		'type' => 'text',	'readonly'=>2, 'data'=>'int',		'width' => '5%'),
-         	'comment_subject' 		=> array('title'=> "subject",		'type' => 'text',			'width' => 'auto', 'thclass' => 'left first', 'writeParms'=>array('size'=>'xxlarge')), // Display name
-         	'comment_comment' 		=> array('title'=> "comment",		'type' => 'textarea',			'width' => '30%', 'readParms' => 'expand=...&truncate=50&bb=1', 'writeParms'=>'size=xxlarge'), // Display name
+			'comment_item_id' 		=> array('title'=> LAN_ITEM." ".LAN_ID,		'type' => 'text',	'readonly'=>2, 'data'=>'int',		'width' => '5%'),
```
Comment (CaMer0n) [inline e107_admin/comment.php]: This will throw a FATAL ERROR in PHP 5.4

**Pull request merged**